cloudinit/tests/test_util.py

```diff
@@ -704,4 +704,72 @@ def test_read_conf_from_cmdline_config(self, expected_cfg, cmdline):
         assert expected_cfg == util.read_conf_from_cmdline(cmdline=cmdline)
 
 
+class TestMountCb:
+    """Tests for ``util.mount_cb``.
+
+    These tests consider the "unit" under test to be ``util.mount_cb`` and
+    ``util.unmounter``, which is only used by ``mount_cb``.
+
+    TODO:
```

Comment:
Can we prefix the TODO before each line so that we can leverage TODO tools or just grep to identify known todos and review before release please?

Reply:
Done.